plugin.video.BlanketFort/default.py

```diff
@@ -0,0 +1,366 @@
+# -*- coding: utf-8 -*-
+
+import urllib
+import urllib2
+import datetime
+import re
+import os
+import xbmcplugin
+import xbmcgui
+import xbmcaddon
+import xbmcvfs
+from BeautifulSoup import BeautifulStoneSoup, BeautifulSoup, BeautifulSOAP
+try:
+    import json
+except:
+    import simplejson as json
+
+addon = xbmcaddon.Addon(id='plugin.video.BlanketFort')
+profile = addon.getAddonInfo('profile').decode('utf-8')
+home = addon.getAddonInfo('path').decode('utf-8')
+icon = xbmc.translatePath(os.path.join(home, 'icon.png'))
+fanart = xbmc.translatePath(os.path.join(home, 'fanart.jpg'))
+
+
+def makeRequest(url):
+
+        try:
+            req = urllib2.Request(url, headers={'User-Agent' : "Magic Browser"}) 
+            response = urllib2.urlopen(req)
+            data = response.read()
+            response.close()
+            return data
+        except urllib2.URLError, e:
+            if hasattr(e, 'code'):
+                xbmc.log('We failed with error code - %s.' % e.code)
+                xbmc.executebuiltin("XBMC.Notification(BlanketFort,We failed with error code - "+str(e.code)+",10000,"+icon+")")

```

Comment:
it would be more 'pythonic' to use the built-in notification dialog here:
http://mirrors.xbmc.org/docs/python-docs/16.x-jarvis/xbmcgui.html#Dialog-notification